packages/components/organisms/f-checkout/src/components/_tests/Selector.test.js

```diff
@@ -413,29 +343,181 @@ describe('Selector', () => {
     });
 
     describe('mounted ::', () => {
-        it('should make a call to `initFulfilmentTime` with the available `fulfilmentTimes`', () => {
+        describe('invokes `initFulfilmentTime`', () => {
             // Arrange
-            const initFulfilmentTimeSpy = jest.spyOn(Selector.methods, 'initFulfilmentTime');
-
-            // Act
-            shallowMount(Selector, {
-                store: createStore(),
-                i18n,
-                localVue,
-                propsData
+            let wrapper;
+            let updateFulfilmentTimeSpy;
+
+            describe('AND when there is fulfilment `times` but no pre-selected fulfilment time', () => {
+                beforeEach(() => {
+                    // Act
+                    wrapper = shallowMount(Selector, {
+                        store: createStore(),
+                        i18n,
+                        localVue,
+                        propsData
+                    });
+
+                    updateFulfilmentTimeSpy = jest.spyOn(Selector.methods, 'updateFulfilmentTime');
+                });
+
+                afterEach(() => {
+                    jest.clearAllMocks();
+                });
+
+                it('should exist', () => {
+                    expect(wrapper.vm.initFulfilmentTime).toBeDefined();
+                });
+
+                it('should make a call to `updateFulfilmentTime`', () => {
```

Comment:
`should make a call to updateFulfilmentTime ... with from & to times passed in`?